docs/source/create/entities/zone-entity.rst

```diff
@@ -0,0 +1,166 @@
+##########################
+Add a Zone Entity
+##########################
+
+A Zone entity enables you to set properties in a specific area. These properties include: flying, ghosting, 
+key lighting, skybox, ambient lighting, haze, bloom, and avatar priorities.
+
+.. contents:: On This Page
+    :depth: 3
+
+-------------------------
+Explanation of Properties
+-------------------------
+
+The number of properties might seem intimidating at first. However, they all serve their own goals and have reason to exist:
+
++------------------------+---------------------------------------------------------------------------------------------+
+| Property               | Description                                                                                 |
++========================+=============================================================================================+
+| Flying Allowed         | Enables or disables flying for people inside the zone. This makes sense for game modes, or  |
```

Comment:
"Enables or disables flying for people inside the zone." -> "Enables or disables flying for avatars inside the zone."

Reply:
The avatar could likely still fly scripts or accessories. The point should be that the person themself cannot.

Reply:
I suppose, that could go either way I sense in terms of how someone will interpret this on the first go. Let's leave it as is then. :D